test/test_pid_reuse.c

```diff
@@ -0,0 +1,122 @@
+/* Test if children processes can reuse PID while NX window is open. */
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/syscall.h>
+#include <assert.h>
+#include <errno.h>
+#include <pthread.h>
+#include "zlib.h"
+
+/* globals shared by all threads */
+z_stream c_stream;
+
+void* comp_file_multith (void *iterations)
+{
+        int err;
+        long *sleep_sec;
+
+	pid_t my_tid = syscall(SYS_gettid);
```

Comment:
Wrong indentation?  This error appears on other lines too.

Reply:
Indeed, thanks for catching that.